fragments/platform/rhel10/validation/unified.ks

```diff
@@ -0,0 +1,6 @@
+# Validate installed packages on RHEL 10 system.
```

Comment:
Are these copied from somewhere? Should they maybe be symlinks to some common files? 

Just asking for possible followup improvements, not nacking this :)

Reply:
Maybe, I think we haven't been using (modifying) this structure enough to start thinking about it yet (copied the stuff so far).
This fragment would be shared between three rhel platforms (rhel8, rhel9, rhel10 ) so maybe we could create a rhel folder as a target for symlinks (or link to rhel8?).
I'd be a bit afraid of trying to be generic at too high cost at this point, but we can track it as a potential follow-up issue.